frontend/test/metabase/scenarios/admin/subscription/payment-failure.cy.spec.js

```diff
@@ -0,0 +1,52 @@
+import { restore, mockSessionProperty } from "__support__/e2e/helpers";
+
+describe("banner", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("Show a banner when the subscription payment status is `past-due`", () => {
+    mockSessionProperty("token-status", {
+      status: "past-due",
+      valid: false,
+      trial: false,
+      features: [],
+    });
+
+    cy.visit("/");
+    cy.contains("We couldn't process payment for your account.");
+    cy.visit(`/admin/`);
+    cy.contains("We couldn't process payment for your account.");
+
+    cy.signInAsNormalUser();
+    cy.visit("/");
+    // Wait for page to load
+    cy.get("header");
```

Comment:
Why do we need this here but not for the previous checks?

Reply:
Because the previous checks are checking for something to exist (so cypress will wait for a bit until it exists), and the check on line 26 is checking for something **not** existing (so cypress will immediately check that it doesn't exist, and not wait for the header to load).

Reply:
nit: We use `cy.icon("gear")` in multiple places for this purpose https://github.com/metabase/metabase/blob/370726bea83aca8b6a3f378bb3e7aaea880d599e/frontend/test/metabase/scenarios/onboarding/setup/user_settings.cy.spec.js#L142, but the existing check looks fine to me with a comment. Maybe `cy.get("header").should("be.visible")`